library/kani/src/lib.rs

```diff
@@ -93,22 +113,43 @@ pub fn any<T: Arbitrary>() -> T {
     note = "This function may return symbolic values that don't respects the language type invariants."
 )]
 #[doc(hidden)]
-pub unsafe fn any_raw<T>() -> T {
-    any_raw_internal()
+pub unsafe fn any_raw<T>() -> T
```

Comment:
Good idea, done.

Reply:
Actually, is rustc smart enough to eliminate compile-time known false asserts? Otherwise, it might make sense to switch it to a `if cfg!() { panic! }` or a `#[cfg()]`.

Reply:
Yes, it is smart enough, both for `if false` and `assert true`.